test/JDBC/input/pg_stat_statements_tsql.mix

```diff
@@ -0,0 +1,238 @@
+-- psql
+CREATE EXTENSION pg_stat_statements WITH SCHEMA sys;
+go
+
+-- psql
+SET sys.pg_stat_statements.track = 'none';
+SELECT sys.pg_stat_statements_reset();
+go
+
+-- tsql
+SELECT dbid, toplevel, query, calls, rows, plans from pg_stat_statements where queryid != 0 ORDER BY query COLLATE "C";
+go
+
+--
+-- simple and compound statements
```

Comment:
Add testcases for:
create/alter/drop type
alter/drop object
rollback/save transaction

Reply:
We are still missing ALTER object statements.